wdl/tasks/CallVariantsPBCCS.wdl

```diff
@@ -268,23 +238,30 @@ workflow CallVariants {
                     is_ccs = true,
                     zones = arbitrary.zones
             }
+
             call VariantUtils.ZipAndIndexVCF as ZipAndIndexPBSV {input: vcf = PBSVslow.vcf }
 
-            call Sniffles.Sniffles as SnifflesSlow {
+            call Sniffles2.SampleSV as sample_snf_slow {
                 input:
                     bam    = bam,
                     bai    = bai,
+                    minsvlen = minsvlen,
+                    sample_id = sample_id,
                     prefix = prefix
             }
-            call Utils.InferSampleName as infer {input: bam = bam, bai = bai}
-            call VariantUtils.FixSnifflesVCF as ZipAndIndexSniffles {input: vcf = SnifflesSlow.vcf, sample_name = infer.sample_name}
+
+            call VariantUtils.ZipAndIndexVCF as ZipAndIndexVCF_slow {
+                input:
+                    vcf = sample_snf_slow.vcf
+            }
         }
     }
 
     output {
-        File? sniffles_vcf = select_first([MergeSnifflesVCFs.vcf, ZipAndIndexSniffles.sortedVCF])
-        File? sniffles_tbi = select_first([MergeSnifflesVCFs.tbi, ZipAndIndexSniffles.tbi])
-
+        File? sniffles_vcf = sample_snf_slow.vcf
```

Comment:
We can drop `sniffles_vcf`.

Reply:
File? sniffles_vcf = ZipAndIndexVCF_slow.vcfgz
(rename alias)